react-intl/index.d.ts

```diff
@@ -227,7 +228,9 @@ declare namespace ReactIntl {
             defaultFormats?: Object;
         }
     }
-    class IntlProvider extends React.Component<IntlProvider.Props, any> { }
+    class IntlProvider extends React.Component<IntlProvider.Props, any> {
+        getChildContext(): InjectedIntlProps;
```

Comment:
Can either be `{intl: InjectedIntl}` or `InjectedIntlProps`. If you look at the code, it returns an object with a single `intl` key.